packages/templates/templates/getting-started-js/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "crawlee-getting-started-js",
+    "version": "0.0.1",
+    "type": "module",
+    "description": "This is an example of a Crawlee project.",
+    "dependencies": {
+        "crawlee": "^3.0.0",
+        "playwright": "*"
+    },
+    "scripts": {
+        "start": "node src/main.js",
+        "build": "tsc",
```

Comment:
Js templates shouldnt have build scripts

Reply:
I copied it from playwright-js and did not check 😅 Will check other templates and delete where appropriate.